pkg/bpf/bpf_linux.go

```diff
@@ -20,6 +21,8 @@ import (
 	"github.com/cilium/cilium/pkg/spanstat"
 )
 
+var keySizeExceptions = []string{"cilium_policy"}
```

Comment:
Could you add a comment to explain when maps are exempt from the key size check? so it is clear why this code exists.